lib/serverMethods.js

```diff
@@ -1,19 +1,19 @@
 const { collection } = require("./database");
 
-async function getCollection(collectionName) {
-  return await collection(collectionName).find({}).toArray();
-}
-
-async function getDataByName(collectionName, dataName) {
-  return await collection(collectionName).findOne({ name: dataName });
-}
-
-async function getDataByOperator(collectionName, dataName, data) {
-  return await collection(collectionName)
-    .find({
-      [dataName]: { $regex: new RegExp(data, "i") },
-    })
-    .toArray();
+async function getCollection({
+  collectionName,
+  dataName = "name",
+  dataValue = "",
+  sortBy = false,
+  order = 1,
+}) {
+  const cursor = collection(collectionName).find({
+    [dataName]: { $regex: new RegExp(dataValue, "i") },
+  });
+  if (sortBy) {
+    cursor.sort({ [sortBy]: order });
+  }
```

Comment:
is there a reason to not order the results? you could control it with the value of order, which is kind of an orderType? for example you want to order for numbers, or letters, you can control that with it?

Reply:
Wasn't sure if there could be a case where we maybe do not need to have it sorted. Thus it was implemented as a query that is accessable from the frontend to be able to just take the sorted or unsorted results. 

From what i read in the [mongoDB documentation](https://docs.mongodb.com/manual/reference/method/cursor.sort/index.html) the `cursor.sort({ field: value })` takes a variable for a field that should be checked and a value(-1 or 1) for descending or ascending order.